packages/xod-project/src/index.js

```diff
@@ -130,8 +132,12 @@ export {
   ensureLiteral,
   migrateBoundValuesToBoundLiterals,
 } from './migrations/boundValuesToBoundLiterals';
+export {
+  default as migrateOldDimensionsToSlots,
+} from './migrations/oldDimensionsToSlots';
```

Comment:
I recommend avoiding words like “old” and “new” in migrations. They are too biased. To be more precise, you might use `unitlessDimensionsToSlots` or just `unitlessToSlots`